store/extensions/GChatSaver/extension.json

```
@@ -0,0 +1,37 @@
+{
+    "title": "GChatSaver",
+    "description": "GChatSaver is an extension for saving all user chats and room information\nSaves all user chats (whispers included)\nSaves Room infos (room name, room owner, room title)\nSaves chat logs in G-Earth folder in a txt file",
+    "authors": [
+      {
+        "name": "Yarkin",
+        "hotel": ".com",
+        "username": "Yarkin",
+        "discord": "y1#1737"
+      }
+    ],
+    "version": "1.0",
+    "categories": ["Fun", "Others"],
+    
+    "source": "https://github.com/iamyarkin/GChat-Saver",
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "G-Python",
+      "version": "1.0"

```

The current version of G-Python is 0.1.6
```suggestion
      "version": "0.1.6"
```